domains/temelkov.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Temelkov",
+        "email": "stanislav.temelkov@hotmail.com"
+    },
+    "record": {
+        "CNAME": "temelkov-tk-2022.vercel.app"
```

Comment:
Vercel is not supported yet (might be soon)

Reply:
@Stef-00012 I Change it. Please merge!